packages/plugin-core/src/commands/DeleteNodeCommand.ts

```diff
@@ -54,17 +65,21 @@ export class DeleteNodeCommand extends BasicCommand<
           Logger.error({ ctx, msg: "error deleting node", error: out.error });
           return;
         }
-        window.showInformationMessage(`${path.basename(fsPath)} deleted`);
+        window.showInformationMessage(
+          formatDeletedMsg({ fsPath, vault: note.vault })
+        );
         return out;
       } else {
         const smod = await DendronClientUtilsV2.getSchemaModByFname({
           fname,
           client,
         });
         await client.deleteSchema(SchemaUtils.getModuleRoot(smod).id);
+        window.showInformationMessage(
+          formatDeletedMsg({ fsPath, vault: smod.vault })
+        );
+        return;
       }
-      window.showInformationMessage(`${path.basename(fsPath)} deleted`);
```

Comment:
Moved this into else statement for clarity since we only would reach this from the `else` statement branch (the if block would exit with `return out`)